### PR TITLE
this plugin now supports fluentd version 0.14 or higher

### PR DESCRIPTION
**Pull request by kentaro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,23 +6,13 @@
...
-
-Imagin you have a config as below:
+## Caveat
 
-fluentd `< 0.12`:
-
-```
-<match test.**>
-  @type extract_query_params
+This plugin, from version 0.1.0, supports fluentd of version 0.14 or higher.
```
Comment (repeatedly) [inline README.md]: Could you add additional line for v0.12 like below?
`If you use fluentd v0.12 or earlier, use v0.0.x versions`

Comment (kentaro) [inline README.md]: Ah, I got it.

**Comment by kentaro** [comment]
Added a note for earlier version of fluentd.

**Comment by repeatedly** [comment]
Thanks!

**Pull request merged**